app/assets/js/rtm-upload-terms.js

```diff
@@ -1,126 +1,140 @@
-
 /**
+ * Check for terms and condition
+ * 
  * Remove Terms Warning Popup,
  * Add Warning Message Before Submit Button
  * By: Malav Vasita <malav.vasita@rtcamp.com>
  */
-if ( 'object' === typeof rtMediaHook ) {
-	rtMediaHook.register( 'rtmedia_js_upload_file', function ( args ) {
-		if ( false === args ) {
-			return args;
-		}
-
-		var terms_conditions_checkbox = jQuery( '#rtmedia_upload_terms_conditions' );
-
-		if ( terms_conditions_checkbox.length > 0 ) {
-			if ( ! terms_conditions_checkbox.is( ':checked' ) ) {
-				rtp_display_terms_warning( terms_conditions_checkbox.parent( '.rtmedia-upload-terms' ), rtmedia_upload_terms_data.message );
-				return false;
-			}
-		}
-
-		return true;
-	} );
-
-	rtMediaHook.register(
-		'rtmedia_js_before_activity_added',
-		function ( args ) {
-			var terms_conditions_checkbox, form;
-			var whats_new_submit = jQuery( '#aw-whats-new-submit' );
-
-			if ( undefined !== args && false !== args && undefined !== args.src && 'activity' === args.src ) {
-				form                      = jQuery( '#whats-new-form' );
-				terms_conditions_checkbox = form.find( '#rtmedia_upload_terms_conditions' );
-			} else {
-				terms_conditions_checkbox = jQuery( '#rtmedia_upload_terms_conditions' );
-			}
-
-			if ( terms_conditions_checkbox.length > 0 ) {
-				terms_conditions_checkbox.removeAttr( 'disabled' );
-
-				if ( false == args ) {
-					whats_new_submit.removeAttr( 'disabled' );
-					whats_new_submit.removeClass( 'loading' );
-					return args;
-				}
-
-				if ( ! terms_conditions_checkbox.is( ':checked' ) ) {
-					whats_new_submit.removeAttr( 'disabled' );
-					whats_new_submit.removeClass( 'loading' );
-					if ( undefined !== args && false !== args && undefined !== args.src && 'activity' === args.src ) {
-						rtp_display_terms_warning( form.find( '.rtmedia-upload-terms' ), rtmedia_upload_terms_data.message );
-					} else {
-						rtp_display_terms_warning( terms_conditions_checkbox.parent( '.rtmedia-upload-terms' ), rtmedia_upload_terms_data.message );
-					}
-					return false;
-				}
-			}
-			return true;
-		}
-	);
-
-	/**
-	 * When Select Attribute for media [ rtmedia-custom-attributes: Add-Ons ] Issue:8,
-	 * This Hook returns false that's why the attributes were not saved,
-	 * Added: 'return true;'
-	 * By: Yahil
-	 */
-	rtMediaHook.register( 'rtmedia_js_after_file_upload', function () {
-
-		var terms_conditions_checkbox = jQuery( '#rtmedia-upload-container #rtmedia_upload_terms_conditions' );
-		if ( terms_conditions_checkbox.length > 0 ) {
-			terms_conditions_checkbox.removeAttr( 'checked' );
-			jQuery( '.rt_alert_msg' ).remove();
-		}
-		return true;
-	});
-
-	rtMediaHook.register( 'rtmedia_js_after_activity_added', function() {
-		var rtmedia_terms_conditions = $( '#rtmedia_upload_terms_conditions' );
-		if ( rtmedia_terms_conditions && rtmedia_terms_conditions.is(':checked') ) {
-			rtmedia_terms_conditions.prop( 'checked', false );
-		}
-	} );
+if ('object' === typeof rtMediaHook) {
```

Comment:
Leave spaces between every ().
https://make.wordpress.org/core/handbook/best-practices/coding-standards/javascript/#arrays-and-function-calls